src/print_event_cb.c

```diff
@@ -621,13 +627,17 @@ print_event_bin(void *cb_cookie, void *data, int size)
 {
 	(void) cb_cookie;
 
+	pthread_spin_lock(&OutputLock);
```

Comment:
Well, looking at bcc, this is where `data` and `size` are coming from:
```
struct {           
    uint32_t size; 
    char data[0];  
} *raw = NULL;
...  
raw = (void *)ptr;
...
reader->raw_cb(reader->cb_cookie, raw->data, raw->size);
```

So, these two things are actually already in a single buffer. We would just need to convince bcc developers to guarantee this in their API.